.dockerignore

```diff
@@ -1,6 +1,8 @@
 .env*
-!.env-sample
+!.env.docker
```

Comment:
We do not need `.env.docker` inside the docker image so it is find ignoring it.
So you may remove the line requesting not to ignore it :)

The `.env.docker` content will/may be passed to the docker container using:
```
docker run --rm -it --env-file .env.docker <...>
```
=> it comes from outside the container and prevents having to pass many `-e ...` chunks in the docker command.

Reply:
Good to know, I have not seen the `--env-file` option before.